helpers/metadata_test.go

```diff
@@ -0,0 +1,95 @@
+package helpers
+
+import (
+	tektonv1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"testing"
+)
+
+var (
+	testObject      = &tektonv1beta1.PipelineRun{}
+	testAnnotations = map[string]string{
+		"a.test/annotation":              "foo",
+		"another.test/annotation-string": "bar",
+	}
+	testLabels = map[string]string{
+		"a.test/label":              "foo",
+		"another.test/label-string": "bar",
+	}
+)
+
+func init() {
+	testObject.SetAnnotations(testAnnotations)
+	testObject.SetLabels(testLabels)
+}
+
+func TestHasAnnotation(t *testing.T) {
+	var tests = []struct {
+		annotation string
+		want       bool
+	}{
+		{"a.test/annotation", true},
+		{"bar", false},
+	}
+
+	t.Logf("Testing that HasAnnotation properly finds object annotations")
```

Comment:
I´d say that all this logs are redundant. A test in Go points to a function most of the times, and the function is (or should) be properly defined. This should be changed by proper documentation for each test function. Generally, for a test output, I'm just interested in seeing which test failed (if any). But this is my personal opinion.

Reply:
@parasense wdyt?